src/bowling.js

```diff
@@ -0,0 +1,110 @@
+class bowling {
+    constructor() {
+
+    this.round = []
+
+    this.bonus = 5 
+
+    this.bonus_two = 10  
+
+    this.number_rounds = 0
+
+    } 
+
+    gameRound(num_one, num_two) {    
+        while (this.number_rounds <= 10) {   
+            this.number_rounds ++
+            if (this.number_rounds <= 10) { 
+                if(num_one + num_two > 10) { 
+                    return "Thats not a realy score, it must be below 10"  
```

Comment:
Good to have a guard clause for this, I didn't think to do that - may be worth having it as an error?